packages/wrangler/src/mtls-certificate/cli.ts

```diff
@@ -0,0 +1,168 @@
+import {
+	uploadMTlsCertificateFromFs,
+	listMTlsCertificates,
+	deleteMTlsCertificate,
+	getMTlsCertificate,
+	getMTlsCertificateByName,
+	type MTlsCertificateResponse,
+} from "../api/mtls-certificate";
+import { withConfig } from "../config";
+import { confirm } from "../dialogs";
+import { logger } from "../logger";
+import { requireAuth } from "../user";
+import type { CommonYargsOptions } from "../yargs-types";
+import type { Argv, BuilderCallback } from "yargs";
+
+interface ConfigArgs {
+	config?: string;
+}
+
+interface UploadMTlsCertificateArgs extends ConfigArgs {
+	cert: string;
+	key: string;
+	name?: string;
+}
+
+interface DeleteMTlsCertificateArgs extends ConfigArgs {
+	id?: string;
+	name?: string;
+}
+
+function uploadMTlsCertificateOptions(
+	yargs: Argv
```

Comment:
```suggestion
	yargs: CommonYargsArgv
```
(imported from `yarg-types.ts`)

Reply:
rebased up and implemented these suggestions - thanks for the tip!